setup.py

```diff
@@ -26,7 +27,7 @@
     if Version == "":
         Version = descr
 except Exception:
-    descr = Version
+    descr = release_version
```

Comment:
```suggestion
    descr = release_version
    Version = release_version
```
Otherwise I think `Version` below would be undefined.